buildpipeline/linux.groovy

```diff
@@ -8,7 +8,16 @@
 
 def submittedHelixJson = null
 
-simpleDockerNode('microsoft/dotnet-buildtools-prereqs:rhel7_prereqs_2') {
+def dockerImageName
```

Comment:
Lets undo the CI changes as they are going to require further changes (i.e. adjustments to the build commands) and it isn't clear to me whether or not we want to include them yet.